pennylane/templates/subroutines/qpe.py

```diff
@@ -200,6 +201,31 @@ def estimation_wires(self):
         """The estimation wires of the QPE"""
         return self._hyperparameters["estimation_wires"]
 
+    def error(self):
+        """The QPE error computed from the spectral norm error of the input unitary operator.
+
+        **Example**
+
+        >>> class CustomOP(qml.resource.ErrorOperation):
+        ...    @property
+        ...    def error(self):
+        ...       return 0.005
+        >>> Op = CustomOP(wires=[0])
+        >>> QPE = QuantumPhaseEstimation(Op, estimation_wires = range(1, 5))
+        >>> QPE.error()
+        0.075
+        """
+        unitary_error = self._hyperparameters["unitary"].error.error
```

Comment:
It is not always the case that the base "unitary" will have an error, in this case we assume it is exact and there is 0 error. Please add a check for this: 

```
base_unitary = self._hyperparameters["unitary"]
if not isinstance(base_unitary, ErrorOperation):
    return SpectralNormError(0.0) 
```

Reply:
Also please add the associated test which computes the error for an operator with no error

Reply:
Added.